UnityProject/Assets/Scripts/Items/ItemAttributesV2.cs

```diff
@@ -182,6 +188,110 @@ public IEnumerable<string> ServerAttackVerbs
 		[SerializeField]
 		private ItemsSprites itemSprites;
 
+		[SyncVar(hook = nameof(FakeItemSync))]
+		public bool IsFakeItem = false;
+
+		[Client]
+		private void FakeItemSync(bool oldValue, bool newValue)
```

Comment:
is best practice to use
FakeItemSync(IsFakeItem, new value)
instead of
IsFakeItem =  new value
then you don't have to worry about it being different across server and client